docs/BOOT_LOADER_INTERFACE.md

```diff
@@ -58,6 +58,17 @@ variables. All EFI variables use the vendor UUID
 * The EFI variable `LoaderEntryDefault` contains the default boot loader entry
   to use. It contains a NUL-terminated boot loader entry identifier.
 
+* The EFI variable `LoaderEntrySysFail` contains the sysfail boot loader entry
+  to use. Sysfail boot entries can be used optional tweaking the automatic
```

Comment:
I am not native, but maybe
`can be used optional tweaking` -> `can be used for optionally tweaking`
??